Userland/Applications/KeyboardMapper/KeyButton.cpp

```diff
@@ -1,8 +1,8 @@
 /*
- * Copyright (c) 2020, Hüseyin Aslıtürk <asliturk@hotmail.com>
- *
- * SPDX-License-Identifier: BSD-2-Clause
- */
+* Copyright (c) 2020, Hüseyin Aslıtürk <asliturk@hotmail.com>
+*
+* SPDX-License-Identifier: BSD-2-Clause
+*/
```

Comment:
Not sure what's going on with these changes.

Reply:
I'm not really sure either - I think I'll blame it on an clang-format error. Anyway, it is now fixed